src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const totalySum = days * 40;
+
+  if (days < 3) {
```

Comment:
[Checklist - paragraph 2] Don't use literals in expressions (HARDCODE). Instead use constants explaining this values.
Same for other ifs.

Also you dont need this `if`, instead use `return` in the end of the function body.